### PR TITLE
feat: Add option to specify prometheus metrics listen address

### PR DESCRIPTION
**Pull request by mmoutama09**
## Description

Add variable listen_address to enable prometheus metrics

## Migrations required

NO

## Verification

plan is joined to the PR
[plan.txt](https://github.com/npalm/terraform-aws-gitlab-runner/files/8631677/plan.txt)

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -131,6 +131,7 @@ locals {
       bucket_name                       = local.bucket_name
       shared_cache                      = var.cache_shared
       sentry_dsn                        = var.sentry_dsn
+      listen_address                    = var.listen_address
```
Comment (kayman-mk) [inline main.tf:134]: question: could this name conflict with the session_server.listen_address setting. If so I suggest to rename it.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,12 +1,13 @@
 repos:
-  - repo: git://github.com/antonbabenko/pre-commit-terraform
+  - repo: http://github.com/antonbabenko/pre-commit-terraform
```
[the remark this answers is not in the document]
Comment (kayman-mk) [reply .pre-commit-config.yaml:2]: Just had the same issue. The `git` protocol is outdated. See #481

File: main.tf (hunk 1 above)
Comment (mmoutama09) [reply main.tf]: i renamed it just in case :)

File: .pre-commit-config.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (npalm) [reply .pre-commit-config.yaml:2]: Should it not be https instead of http?

**Review by npalm — CHANGES_REQUESTED** [review]
@mmoutama09 thanks for your contribution, sorry for keep you wainting. Just a small remark on the pre-commit. Can you update it to https?

**Comment by npalm** [comment]
Just merged https://github.com/npalm/terraform-aws-gitlab-runner/pull/481 fixing the submodule

**Review by npalm — APPROVED** [review]
All good, just synced with other PRs

**Pull request merged**